src/engraving/libmscore/undo.cpp

```diff
@@ -292,11 +292,7 @@ UndoStack::UndoStack()
 
 UndoStack::~UndoStack()
 {
-    size_t idx = 0;
-    for (auto c : list) {
-        c->cleanup(idx++ < curIdx);
-    }
-    DeleteAll(list);
```

Comment:
Where has `DeleteAll(list);` gone in `UndoStack::clear()`? It looks like we would still need it to prevent memory leaks, don't we?